src/Components/SignupLogin/SignUp.js

```diff
@@ -29,7 +32,12 @@ const SignUp = () => {
 
           <Form.Group controlId="passWord">
             <Form.Label>Password</Form.Label>
-            <Form.Control type="password" placeholder="Password" />
+            <InputGroup>
+              <Form.Control type={showPassword ? 'text' : 'password'}  placeholder="Password" />
+              <InputGroup.Append>
+                <Button variant="secondary" onClick={handleShowPassword}>{showPassword ? 'Hide' : 'Show'}  </Button>
```

Comment:
Use react-icon (BsEye) instead of Button.

Reply:
done